app/api/__init__.py

```diff
@@ -162,7 +159,7 @@
 
 # tax
 api.route(TaxList, 'tax_list', '/events/<int:id>/tax', '/events/<identifier>/tax')
-api.route(TaxDetail, 'tax_detail', '/tax/<int:id>', '/events/<int:event_id>/tax')
+api.route(TaxDetail, 'tax_detail', '/tax/<int:id>')
```

Comment:
@niranjan94 @SaptakS 
I think that if we want to make list post only and detail get only, then we will have to change one of the endpoints name to something else like name POST list method to `events/<event_id>/taxes` just as we did with speakers-calls. Is there any other way that we can do POST only list and GET only detail with same endpoint: `events/<event_id>/tax`?
( `events/<int:id>/tax` is to be changed to `events/<int:event_id>/tax` )